public/js/favorite.js

```diff
@@ -0,0 +1,54 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+function deleteFavorite(room) {
+  fetch(`/favorites/${room}`, { method: 'DELETE' })
+    .then(async res => {
+		switch(res.status) {
+		  case 201:
+			location.reload()
+        	break
+		  case 401:
+        	displayMessage('danger', UNAUTHORIZED_MESSAGE)
+        	break
+		  case 403:
+        	displayMessage('danger', FORBIDDEN_MESSAGE)
+        	break
+		  case 404:
+        	data = await res.json()
+        	displayMessage('danger', data.message)
+        break
+		  }
+    })
+    .catch(err => displayMessage('danger', err))
+}
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+function addFavorite(room) {
+	if (!room) {
+		displayMessage('danger', err)
+  	} else {
+		fetch('/favorites', { 
+			method: 'POST',  
+			headers: {
+      			'Content-Type': 'application/json'
+    		},
+			body: JSON.stringify({ room }), 
+		})
+    .then(async (res) => {
+		  switch (res.status) {
+		  	case 201:
+			    location.reload()
+          break
+		  	case 401:
+          displayMessage('danger', UNAUTHORIZED_MESSAGE)
+					break
+				case 403:
+        	displayMessage('danger', FORBIDDEN_MESSAGE)
+        	break
+		  	case 404:
+        	data = await res.json()
+        	displayMessage('danger', data.message)
+        	break
+		  }
+      })
+      .catch((err) => displayMessage('danger', err))
+  }
+}
```

Comment:
Could you please correct the indentation in this file? Especially here.
![Screenshot 2020-09-08 16 59 17](https://user-images.githubusercontent.com/60111772/92493970-6f037300-f1f5-11ea-850d-42a57b3c82c0.png)

We like to indent using two spaces.
Consistency (even with indentation) is very nice for the developer's eyes. 😄

Reply:
Sorry for that. My VS code is automatically indenting with tabs, and resets the settings after every restart. I will run a lint with --fix option. That will do it.